packages/calcite-components/src/components/swatch-group/swatch-group.scss

```diff
@@ -0,0 +1,26 @@
+:host {
+  display: flex;
+}
+
+.container {
+  display: flex;
+  inline-size: 100%;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: theme("spacing.2");
+}
+
+::slotted(calcite-swatch) {
+  flex: none;
+}
+
+:host([scale="s"]) .container {
+  gap: theme("spacing.1");
+}
+
+:host([scale="l"]) .container {
+  gap: theme("spacing.3");
```

Comment:
If we want to keep the existing spacing from a large scale Color Picker, we should be using an 8px gap for `m` and `l`, and a 4px gap for `s`.